### PR TITLE
Attempt 2 to solve the issue of sssom.yaml not showing in site-packages after pip install

### PR DESCRIPTION
**Pull request by hrshdhgd**
Relevant to issue #183 and follow-up to #184

### CODE REVIEW

**Comment by hrshdhgd** [comment]
fixed #181 fixed #183

File: setup.py [hunk 1]
```diff
@@ -5,4 +5,4 @@
 import setuptools
 
 if __name__ == "__main__":
-    setuptools.setup()
+    setuptools.setup(include_package_data=True)
```
Comment (cthoyt) [inline setup.py:8]: put this in the `setup.cfg` under the `[options]` header. See https://cthoyt.com/2020/06/03/how-to-code-with-me-organization.html for more information on how to do packaging

File: tests/test_cli.py [hunk 2]
```diff
@@ -73,12 +73,13 @@ def run_successful(self, result: Result, test_case: SSSOMTestCase) -> None:
 
     def run_convert(self, runner: CliRunner, test_case: SSSOMTestCase) -> Result:
         """Run the convert test."""
+        fmt = "owl"
```
Comment (cthoyt) [inline tests/test_cli.py:76]: relevant?

Comment (hrshdhgd) [reply tests/test_cli.py:76]: no ... I usually don't add irrelevant changes but this was minor, so thought of just pushing it.

Comment (cthoyt) [reply tests/test_cli.py:76]: kk :shipit:

File: setup.py (hunk 1 above)
Comment (hrshdhgd) [reply setup.py]: Done and tested. Works as expected!

**Review by cthoyt — APPROVED** [review]
I'm a little concerned that there's no specific git commit hash that corresponds to the releases, but that's an issue to solve elsewhere

**Pull request merged**